install/class.Setup.php

```diff
@@ -263,6 +213,16 @@ public function __invoke($params)
 
         $serviceManager = $installator->getServiceManager();
 
+        if (!isset($parameters['configuration']['generis']['persistences'])) {
+            throw new InvalidArgumentException('Your config should have a \'persistence\' key under \'generis\'');
+        }
+        $persistences = $parameters['configuration']['generis']['persistences'];
+        if (isset($persistences['default'])) {
```

Comment:
this can be set to '' and isset will return true
```
$persistences['default'] = '';

 if (isset($persistences['default'])) {
     echo  'breaking somthing';
 }
```
```suggestion
         if (!empty($persistences['default'])) {
```

Reply:
This is not a validation but a test to see if persistences are defined as a service or as direct configuration parameters:
if $persistences['default'] is set => direct persistence configuration parameters
if $persistences['type'] is set => full service configuration
if neither is set => PROBLEM

else there would be many more tests required: Is is an array, does it contain a key db_driver and so on